metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-puppet_operational_dashboards",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Please revert this back, we update the metadata version by the release pipeline so this is unnecessary.

Reply:
Done.